main.cpp

```diff
@@ -117,16 +112,24 @@ int main(int argc, char *argv[])
     app.installEventFilter(eventFilter);
 
     QCommandLineParser parser;
+    QCommandLineOption logPathOption(QStringList() << "l" << "log",
```

Comment:
How about using `--log-file` like all the other CLI apps, as opposed to the current implementation where the supplied path is assumed to be a directory? I think this is less confusing.